x-pack/plugins/security/public/account_management/index.ts

```diff
@@ -6,6 +6,3 @@
  */
 
 export { accountManagementApp } from './account_management_app';
-
-export type { ChangePasswordProps } from './change_password';
-export type { PersonalInfoProps } from './personal_info';
```

Comment:
I can't delete these unfortunately since they're used by `/x-pack/plugins/security/public/ui_api/components.tsx`. 

Does anyone know why this was added and what the use case is?

Reply:
Hi folks, thanks for the ping. I am now currently in another team, so pinging folks that work on our plugin.

@elastic/ent-search-application-frontend could you please check the questions above?

Reply:
I filed [an issue](https://github.com/elastic/enterprise-search-team/issues/1648) for the Enterprise Search team to explore either updating to the new components, or leaving the legacy components in place. Please let me know if the decision is blocking anyone; I expect/hope that continuing to use the legacy components for now will not block progress in the rest of the stack.